pkg/system/utils_unix.go

```diff
@@ -26,7 +27,8 @@ func resolveBinary(binname string) (string, error) {
 	return "", fmt.Errorf("Binary %q does not resolve to a binary of that name in $PATH (%q)", binname, resolvedPath)
 }
 
-func execCmd(cmd, args string) ([]byte, error) {
+// ExecCmd calls exec.Command and returns the CombinedOutput.
+func ExecCmd(cmd, args string) ([]byte, error) {
```

Comment:
The file is only these 2 functions, right? so yes I think it would all move to `pkg/system/user`.